pkg/ingester/user_state.go

```diff
@@ -292,12 +296,21 @@ outer:
 			}
 		}
 
-		err := callback(ctx, fp, series)
+		err := append(ctx, fp, series)
```

Comment:
`append` is a well-known function in Go; I think it's too confusing to re-use the name here